dev/com.ibm.ws.transaction.cdi/bnd.bnd

```diff
@@ -55,4 +55,4 @@ Export-Package: \
 	../build.sharedResources/lib/junit/old/junit.jar;version=file, \
 	com.ibm.ws.logging;version=latest
 
-jakartaeeMe: true
+test.project: true
```

Comment:
Not 100% sure that this is the right thing to put here ... but it does mean that the `com.ibm.ws.transaction.cdi` bundle will never be copied into wlp/lib, which would be bad!

Reply:
It looks like other projects which contain product code but don't themselves get included in the image use `publish.wlp.jar.disabled: true`.

I would worry that `test.project: true` may have other side effects.

Reply:
Might also be worth a comment somewhere if you're sucking out the classes into both of the auto-feature bundles.

Reply:
```suggestion
# This bundle is not shipped itself. Instead, the classes are included in one of two other bundles, depending on the
# cdi version being used: io.openliberty.transaction.internal.cdi12 or io.openliberty.transaction.internal.cdi20
publish.wlp.jar.disabled: true
```